cmd/nginx-ingress/main.go

```diff
@@ -64,6 +64,9 @@ func main() {
 	parseFlags()
 	parsedFlags := os.Args[1:]
 
+	buildOS := os.Getenv("BUILD_OS")
+	fmt.Println("BUILD OS:", buildOS)
```

Comment:
```suggestion
```

Reply:
Thanks, just noticed that, it was a debug.